meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/base_dashboard_project.html

```diff
@@ -54,3 +54,8 @@
         </div>
     </div>
 {% endblock %}
+
+
+{% block extra_js %}
+    <script src="{% static 'js/unload_warning.js' %}"></script>
```

Comment:
maybe we could just include the  script tag in the body/dashboard_content to ensure it is always loaded. currently templates inheriting from base_dashboard_project.html have to remember to call block.super

Reply:
I really dislike having JS **somewhere** in the middle of `<body>`. I know there are possibly already occurrences where we do this and I know loading it in `<head>` isn't better, but I'd rather not make it worse.

Reply:
okay, then i'll merge now :)